src/main/java/EmailNotification.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Scanner;
+
+public class EmailNotification implements Notifications {
+    @Override
+    public void sendNotification(String message) {
```

Comment:
this class should only send message to target email. Method signature should be something like : sendNotification(String emailFrom, String emailTo, String message)